src/features/metadata/components/EnvBuilds.tsx

```diff
@@ -9,12 +9,14 @@ interface IData {
   currentBuildId: number;
   selectedBuildId: number;
   builds: IBuild[];
+  mode: "create" | "read-only" | "edit";
```

Comment:
Are these the same modes we use elsewhere? Just making sure these are consistent throughout

Reply:
Yes, those are the three consistent states